src/backend/app/projects/project_routes.py

```diff
@@ -24,7 +26,11 @@
 
 
 @router.delete("/{project_id}", tags=["Projects"])
-def delete_project_by_id(project_id: int, db: Session = Depends(database.get_db)):
+def delete_project_by_id(
+    project_id: uuid.UUID,
+    db: Session = Depends(database.get_db),
+    AuthUser=Depends(login_required),
```

Comment:
This is missing a param name:

`user: AuthUser = Depends(login_required)`

as auth user is just the param type

Reply:
@spwoodcock  I have just fixed it by adding the missing parameter name. Thanks for pointing that out!